logging.sh

```diff
@@ -1,9 +1,13 @@
 # Log output automatically
 LOGDIR="$(dirname $0)/logs"
-if [ ! -d "$LOGDIR" ]; then
-    mkdir -p "$LOGDIR"
+if [[ -z "${LOGPREFIX:-}" ]]; then
```

Comment:
want to document LOGPREFIX somewhere?

Reply:
I'm trying to break #959 up to make it easier to review, but maybe I should have left this one in that series.

I don't know that anyone would actually use this variable, but the new multicluster.sh script uses it so that all of the logs for a given batch job end up in the same subdirectory.

Reply:
OK, not a big deal